frontend/src/libs/components/map/map-inner-component/map-inner-component.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useRef } from 'react';
```

Comment:
import from '~/libs/hooks/hooks.js';
we need to make import from the library only in 1 place